tests/integration/test_wazuh_db/data/agent_messages.yaml

```diff
@@ -37,8 +41,11 @@
                                         "reference":"03c06c4f118618400772367b1cf7e73ce0178e02",
                                         "type":"PACKAGE",
                                         "status":"VALID",
-                                        "check_pkg_existence":false}'
-    output: ['ok {"action":"UPDATE","status":"SUCCESS"}']
+                                        "check_pkg_existence":false,
+                                        "severity":"Medium",
+                                        "cvss2_score":5.0,
+                                        "cvss3_score":6.1}'
+    output: ['ok {"action":"UPDATE"}']
```

Comment:
This is not related to the IT itself, but we want to receive an output without the STATUS? since we are already getting that information when we `insert` a package.

Reply:
You are right, we should fix this situation in the code, and then update the test